lib/pliny/commands/generator/schema.rb

```diff
@@ -38,34 +46,38 @@ def warn_legacy
         @warned_legacy = true
       end
 
+      def schema_path
+        if legacy?
+          "./docs/schema"
+        else
+          "./schema"
+        end
+      end
+
       def schema_json_path
         if legacy?
-          "./docs/schema.json"
+          "#{schema_path}.json"
```

Comment:
Shouldn't this be `"#{schema_path}/schema.json"`? Similar goes for the `meta.json`.